### ISSUE TITLE
Datepicker in bootstrap-modal

### ISSUE THREAD

**Issue by piwo1984** [eternicode/bootstrap-datepicker#325]
Hi,

I'm using the bootstrap-modal extension from here https://github.com/jschr/bootstrap-modal/
When I attach a datepicker to an input within a modal the picker is positioned far away under the input. I've extended your 'place'-method. Maybe you need it:

```javascript
place : function() {
            if (this.isInline)
                return;
            var zIndex = parseInt(this.element.parents().filter(function() {
                return $(this).css('z-index') != 'auto';
            }).first().css('z-index')) + 10;
            var offset = this.component ? this.component.offset() : this.element.offset();
            var height = this.component ? this.component.outerHeight(true) : this.element.outerHeight(true);
            var isInModal = this.element.parents('.modal').length;
            var top = offset.top + height;
            if(isInModal) {
                top = top - 40;
            }
            this.picker.css({
                top : top,
                left : offset.left,
                zIndex : zIndex
            });
        },
```

**Comment by giaule56** [comment]
I have the same proble like this 
![errordatepicker](https://cloud.githubusercontent.com/assets/7200212/5593983/ed8a802a-9264-11e4-9044-3340684b4f3e.png)

I cannot find the solution

**Comment by hebbet** [comment]
@giaule56 did you upgrade to 1.3.1?

**Comment by acrobat** [comment]
I think the issue @giaule56 will be fixed in 1.4.0 with the new container option #918 and the placement fixes from #1227

**Comment by acrobat** [comment]
Fixed in #1227
